engine/table/src/main/java/io/deephaven/engine/table/impl/select/WhereFilter.java

```diff
@@ -20,7 +20,6 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
-import java.util.stream.Collectors;
```

Comment:
Since we're cleaning up, want to hit the unused import of:
`import io.deephaven.engine.table.impl.BlinkTableTools;`
 in `QueryTable`?